tests/conftest.py

```diff
@@ -35,20 +38,36 @@ def setup_rand() -> Random:
     return Random(231)
 
 
+@fixture
+def setup_landscape():
```

Comment:
rename with 'setup_landscape_one_holder'
the same is for next fixture - 'with' don't need here